### PR TITLE
windows improvements

### PR DESCRIPTION
**Pull request by Trass3r**
* adds utils to CMake build
* adds a basic natvis for debugging
* tries to fix some issues with apps

### CODE REVIEW

File: apps/camera_pipe/process.cpp [hunk 1]
```diff
@@ -1,10 +1,37 @@
-#include <arpa/inet.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
+
```
Comment (abadams) [inline apps/camera_pipe/process.cpp]: There's another PR out that will simplify the timing here much more using C++11 chrono stuff. I'd revert the changes in this file.

**Comment by abadams** [comment]
@delcypher , I'd appreciate your comments on this.

File: apps/camera_pipe/process.cpp (hunk 1 above)
Comment (Trass3r) [inline apps/camera_pipe/process.cpp]: Sure, just experimental.

**Comment by Trass3r** [comment]
Yeah if there was a way to specify env PATH for that add_custom_command we could let them be in the other folders.

**Comment by Trass3r** [comment]
The new QueryPerformanceFrequency stuff didn't work either (C2733). Tried to fix it.

**Comment by Trass3r** [comment]
Rebased and fixed the casts.
